estate/__manifest__.py

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "real estate",
+    "description": "real estate renting management system",
+    "category": "Tutorials",
+    "version": "1.1",
+    "application": True,
+    "data": ["security/ir.model.access.csv", "views/views.xml", "views/menus.xml"],
```

Comment:
Please split your data files (same comment for as many lists as possible):
```suggestion
    "data": [
        "security/ir.model.access.csv",
        "views/views.xml",
        "views/menus.xml",
    ],
```

If someone has to add a file, it will be way easier and keep a clear overview (with git history) of the changes.